tests/unit/sagemaker/cli/compatibility/v2/modifiers/test_training_input.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import pasta
+import pytest
+
+from sagemaker.cli.compatibility.v2.modifiers import training_input
+from tests.unit.sagemaker.cli.compatibility.v2.modifiers.ast_converter import ast_call, ast_import
+
+
+@pytest.fixture
+def constructors():
+    return (
+        "sagemaker.session.s3_input(s3_data='s3://a')",
```

Comment:
need to add:

* `sagemaker.inputs.s3_input`
* `inputs.s3_input`
* `session.s3_input`

(the last two are if someone does `from sagemaker import inputs/session`